debezium-connector-mysql/src/main/java/io/debezium/connector/mysql/BinlogReader.java

```diff
@@ -490,6 +504,9 @@ protected void handleInsert(Event event) throws InterruptedException {
             logger.debug("Skipping previously processed row event: {}", event);
             return;
         }
+        if (ignoreDmlEventByGtidSource) {
+            return;
```

Comment:
Unless you're using the Docker images, then you just set the `LOG_LEVEL` environment variable to `DEBUG` (or `TRACE` or any other valid level).